Package.swift

```diff
@@ -39,8 +43,10 @@ let package = Package(
                 .product(name: "SwiftUIIntrospect", package: "swiftui-introspect"),
                 .product(name: "ExyteMediaPicker", package: "MediaPicker"),
                 .product(name: "FloatingButton", package: "FloatingButton"),
-                .product(name: "ActivityIndicatorView", package: "ActivityIndicatorView")
-            ]
+                .product(name: "ActivityIndicatorView", package: "ActivityIndicatorView"),
+                .product(name: "GiphyUISDK", package: "giphy-ios-sdk")
```

Comment:
The 2 main options for gif 3rd party API are Tenor and Giphy. Tenor is owned by google, whatsapp is owned by Meta. The Giphy API works well is free and is used in WhatsApp so I chose to use Giphy for the sticker / gif input